scripts/NuGetUtilities.psm1

```diff
@@ -17,8 +17,8 @@ $NugetExePath = "nuget"
 if ($ENV:OS) {
     $NugetExePath = "$RepoRoot\.nuget\NuGet.exe"
 }
-$NuGetPackageRoot = Join-Path $SourceRoot "packages"
-$NuGetSamplesPackageRoot = Join-Path (Join-Path $SourceRoot "Samples") "packages"
+$NuGetPackageRoot = "$RepoRoot\bld\packages\"
+$NuGetSamplesPackageRoot = "$RepoRoot\src\Samples\packages\"
```

Comment:
Can you explain the reason of the change,
1. I believe packages is not under bld
2. would Join-Path syntax better? The current code may works in Linux but with Join-Path the benefit is don't have to think or test if \ actually works in Linux and macOS

Reply:
OK, great note. Couldn't get the join-path syntax working previously and hard-coded this change. Let me revert and see whether everything still works.

Thanks again, this occurred to me previously (thanks to your advocacy on this issue) and then I forgot about the potential issue again. :)